vault/resource_pki_secret_backend_role_test.go

```diff
@@ -307,6 +308,7 @@ resource "vault_pki_secret_backend_role" "test" {
   key_type                           = "rsa"
   key_bits                           = 2048
   ext_key_usage                      = []
+  ext_key_usage_oids                 = []
```

Comment:
Looks like [TestPkiSecretBackendRole_basic](https://github.com/hashicorp/terraform-provider-vault/blob/e3f8bc994c895ec7a7538cee9cf0b30cf9368975/vault/resource_pki_secret_backend_role_test.go#L129) is missing `resource.TestCheckResourceAttr(...) entry` for `ext_key_usage_oids` since this belongs to https://github.com/hashicorp/terraform-provider-vault/blob/e3f8bc994c895ec7a7538cee9cf0b30cf9368975/vault/resource_pki_secret_backend_role_test.go#L341